### PR TITLE
Releasing app 0.7

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

releasing App 0.7

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda

### CODE REVIEW

File: docs/source-app/index.rst [hunk 1]
```diff
@@ -232,6 +233,7 @@ Keep Learning
    Run an App on the cloud <workflows/run_app_on_cloud/index>
    Run Apps on your cloud account (BYOC) <workflows/byoc/index>
    Run work in parallel <workflows/run_work_in_parallel>
+   Save files <glossary/storage/drive.rst>
```
Comment (edenlightning) [inline docs/source-app/index.rst:236]: remove

File: docs/source-app/workflows/build_command_line_interface/cli_client.rst [hunk 2]
```diff
@@ -80,7 +80,7 @@ To see a list of available commands:
     --help     Show this message and exit.
 
     Lightning App Commands
-        run notebook Description
+        run notebook Run a Notebook.
```
Comment (edenlightning) [inline docs/source-app/workflows/build_command_line_interface/cli_client.rst:83]: remove

File: docs/source-lit/index.rst [hunk 3]
```diff
@@ -76,6 +77,7 @@ Welcome to ⚡ Lightning Apps
    Run an App on the cloud <workflows/run_app_on_cloud/index>
    Run Apps on your cloud account (BYOC) <workflows/byoc/index>
    Run work in parallel <workflows/run_work_in_parallel>
+   Save files <glossary/storage/drive.rst>
```
Comment (edenlightning) [inline docs/source-lit/index.rst:80]: remove

**Comment by Borda** [comment]
@nohalon @edenlightning mind checking again :otter:

**Review by edenlightning — APPROVED** [review]

**Review by nohalon — APPROVED** [review]
Thanks jirka!!

**Comment by tchaton** [comment]
@Borda Could you include all the commits from the master? I don't see those commits: https://github.com/Lightning-AI/lightning/pull/14995 and https://github.com/Lightning-AI/lightning/pull/15187

**Comment by Borda** [comment]
> Could you include all the commits from the master? I don't see those commits: #14995 and #15187

not really; this is specifically requested for secrets, so your components will follow in a few days, or we can make RC from master

**Review by hhsecond — APPROVED** [review]
LGTM

**Review by lantiga — APPROVED** [review]
LGTM

**Comment by Borda** [comment]
ok, this shall be `rebase & merge` but never mind :otter:

**Pull request merged**